noxfile.py

```diff
@@ -16,3 +16,8 @@ def install_deps(session):
 def build(session):
```

Comment:
how about we call this `build-live`

you can also use hyphens in the name by following a pattern like this:

https://github.com/2i2c-org/infrastructure/blob/b910e65544d35b354122b264037bc5a9fd6e1b61/noxfile.py#L17-L19

Reply:
Makes sense and thanks for the link! I though about doing it this way but was not sure if it was fine to change it since it is also used in the readme.